Sources/Ecc/StdH.h

```diff
@@ -21,7 +21,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <stdarg.h>
 #include <math.h>
 
-#if !PLATFORM_MACOSX
+#if !defined(PLATFORM_MACOSX) && !defined(PLATFORM_FREEBSD)
```

Comment:
maybe make this `#ifdef PLATFORM_WIN32`?  
While glibc has `malloc.h` and doesn't complain if you use it,  according to the manpage `stdlib.h` should be used, like on FreeBSD and OSX.